accesscontroltool-apps-package/src/main/jcr_root/apps/cq/core/content/nav/tools/security/actool/_rep_policy.xml

```diff
@@ -5,12 +5,4 @@
         jcr:primaryType="rep:DenyACE"
         rep:principalName="everyone"
         rep:privileges="{Name}[jcr:read]"/>
-    <allow1
-        jcr:primaryType="rep:GrantACE"
-        rep:principalName="admin"
-        rep:privileges="{Name}[jcr:read]"/>
-    <allow2
-        jcr:primaryType="rep:GrantACE"
-        rep:principalName="administrators"
-        rep:privileges="{Name}[jcr:read]"/>
```

Comment:
Good finding, it would maybe be good to leave a hint here that admin and administrator have access by default (in case somebody wants to understand this later).

Reply:
Well we should rather extend the documentation for ACtool in general that AEM never evaluates ACEs for admins!

Reply:
In the best case even in ACTool we would emit a warn in case you try to set ACEs for admins (default names: admin and administrators)